gpumon/gpumon_client.ml

```diff
@@ -0,0 +1,31 @@
+(*
+ * Copyright (C) Citrix Systems Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation; version 2.1 only. with the special
+ * exception on linking described in file LICENSE.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *)
+
+open Gpumon_interface
+open Xcp_client
+
+let json_url () = "file:" ^ json_path
```

Comment:
`json_path` is set in `gpumon/gpumon_interface.ml`:
`let json_path = "/var/xapi/" ^ service_name ^ ".json"`
so that `json_url` ends up as `file:/var/xapi/gpu.json` (as a side note, i'm changing the service_name to `gpumon` instead of `gpu`). I'm not sure if `file:/...` also works as the recommended `file://...`, but memory/memory_interface.ml does the same thing, and my assumption was that it worked.